docs/INSTALLATION.md

```diff
@@ -0,0 +1,89 @@
+- Amplified speaker ouputs **CAN NOT** be bridged, attempting to do so will damage the amplifier and void the warranty
+
+### Runnning wire in walls and ceilings
+- Most electric codes require the use of CL2 rated speaker wire for in-wall installations. Please refer to local building codes for more details.
+- Avoid running speaker wire next to AC power wires as much as possble to reduce noise.
+- When necessary, cross AC wires at 90 degree angles to avoid introducing any noise into the speakers.
+
+### More Info
+
+Much more information on speaker selection and installation can be the found in AmpliPro’s online documentation and question forums found at the end of this section.
+
+## Preamp Outputs
+Each of AmpliPro's zone outputs have a corresponding line-level preout output pair. These outputs can be used to connect powered subwoofers and other active/powered speakers.
+
+The volume output of a preout is controlled by the corresponding speaker output. This allows a connected subwoofer to be controlled propoertionally to the speakers output in the same zone.
+
+## Audio Inputs
+Each of the stereo RCA inputs can be connected to a different audio source, such as the output of a TV or a record player.
+Expansion Units
+To increase the number of zones you can add expansion units to your system. You can add up to 5 additional zone expander units to a single AmpliPro main unit. Each expansion unit adds 6 zones or pairs of speakers. Zones attach to main units using the CHAIN IN/OUT connectors on the rear panel.These are included with each Zone Expander.
+
+## Network connection
+Connect an RJ45 cable to the Ethernet port on the main unit. Connect the other end to a port on your main network (likeliy on your router). The unit's IP address is configured by DHCP by default.
+
+## Power
+**WARNING!! VERIFY THAT THE VOLTAGE OF YOUR WALL OUTLET MATCHES THE VOLTAGE LISTED NEAR THE POWER INLET.**
+
+AmpliPro ships preconfigured for the typical mains voltage in your region, either 120V mains power or 230V mains; you can see what mode it was set to before shipment based on which box is checked near the power inlet on the unit. If your unit is preconfigured incorrectly for your region, several changes need to be made to the unit to connect to alternative mains:
+- Use an appropriate voltage power cable (AmpliPro only ships with the cable rated for the region)
+- Change the internal power supply fuse located in the IEC socket (located just above the power inlet on the exterior of the AmpliPro)
+- Switch the internal power supply as needed (See below)
+
+Internally, there is a red switch on the side of the power supply that can be toggled for 230V. The easiest way to get at this switch is to take off the top cover. The power supply switch can potentially be accessed through the vent slots on the right side of the unit as well. Here are some pictures to help clarify the process:
```

Comment:
absolutly can not tell people in the manual to stick pokey things through the vent holes into the box

Reply:
Idk, telling customers to stick pokey metal things at our power supply seems pretty rad to me

This section was vestigial from the original draft, I assume the idea was that our professional hobbyist DIY customer base would uninstall shit properly but we def can't have this anymore, you're right